pysoa/server/action/status.py

```diff
@@ -147,6 +168,44 @@ def run(self, request):
 
         return status
 
+    def _check_client_settings(self, request):
+        """
+        This method checks any client settings configured for this service to call other services, calls the `status`
+        action of each configured service with `verbose: False`, adds that diagnostic information, and reports any
```

Comment:
could you add here that `verbose: False` guarantees no further recursive calls?

Reply:
Good thinking. Done.